packages/aws-cdk-lib/aws-eks/test/cluster.test.ts

```diff
@@ -2209,6 +2209,42 @@ describe('cluster', () => {
         Manifest: JSON.stringify([sanitized]),
       });
     });
+    test('trn1 instances are supported', () => {
+      // GIVEN
+      const { stack } = testFixtureNoVpc();
+      const cluster = new eks.Cluster(stack, 'Cluster', { defaultCapacity: 0, version: CLUSTER_VERSION, prune: false });
+
+      // WHEN
+      cluster.addAutoScalingGroupCapacity('TrainiumInstances', {
+        instanceType: new ec2.InstanceType('trn1.2xlarge'),
+        minCapacity: 1,
+      });
+      const fileContents = fs.readFileSync(path.join(__dirname, '..', 'lib', 'addons', 'neuron-device-plugin.yaml'), 'utf8');
+      const sanitized = YAML.parse(fileContents);
+
+      // THEN
+      Template.fromStack(stack).hasResourceProperties(eks.KubernetesManifest.RESOURCE_TYPE, {
+        Manifest: JSON.stringify([sanitized]),
+      });
+    });
+    test('trn1n instances are supported', () => {
```

Comment:
This unit test seems to be added twice

Reply:
they look close but one is `trn1` and the other is `trn1n`